data/tips.cfg

```diff
@@ -14,7 +14,7 @@
 [/tip]
 [tip]
     text= _ "Don’t neglect to earn your leader experience! You need to keep him safe, but if you coddle him too much he may be too weak to survive future battles."
-    source= _ "<i>― Great Mage Delfador, 516YW</i>"
+    source= _ "<i>― Great Mage Delfador, 517YW</i>"
```

Comment:
This isn't the correct date either - the Timeline Wiki page wasn't updated for #4320.